app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -375,6 +383,17 @@ def delete_threatmetrix_response_body(form_response)
       threatmetrix_result.delete(:response_body)
     end
 
+    def threatmetrix_failure_reason(success, result)
+      return nil if success
+
+      tmx_summary_reason_code = result.dig(
+        :response_body,
+        :tmx_summary_reason_code,
+      ) || ["Unexpected ThreatMetrix review_status value: #{result[:review_status]}"]
```

Comment:
issue: `review_status` may not have a value if the TMX API request fails and returns an exception in the response body. The fallback message prefix when this happens has caused quite a bit of confusion for non-Ada/new oncall folks in the past.

I would suggest having different error messages that depended on the presence of `:review_status` and `:exception` in result.

Reply:
this felt a bit too narrow to me as well, but i am not sure what those exceptions would look like, and there aren't enough details in the tests, so i don't think i can confidently do this. 

however, i will happily update the fallback to something more generic to avoid confusion!

Reply:
Synced out-of-band and the path forward is to just return a generic "something went wrong" message if there is no exception message and `:tmx_summary_reason_code` doesn't exist.

Note for posterity: 

If the request to LN completely fails, `:response_body` will not have any of the proprietary fields like `:tmx_summary_reason_code` since we build a dummy response body just to contain our internal exception message for observability purposes. Additionally, no `idv_threatmetrix_response_body` analytics event will be emitted since we weren't able to actually complete the request.

Reply:
thanks so much! added in https://github.com/18F/identity-idp/pull/12099/commits/82d383701b7f312bd2b68ca39b19c286e4ea714a -- let me know if this looks good @lmgeorge